ivy/functional/ivy/gradients.py

```diff
@@ -434,21 +463,22 @@ def adam_update(
 
 
 @to_native_arrays_and_back
+@handle_nestable
 def lamb_update(
-    ws,
-    dcdws,
-    lr,
-    mw_tm1,
-    vw_tm1,
-    step,
+    ws: Union[ivy.Array, ivy.NativeArray],
+    dcdws: Union[ivy.Array, ivy.NativeArray],
```

Comment:
`dcdw` not `dcdws`, we assume singular and then handle multiple via nestable property.